tempesta_fw/http.c

```diff
@@ -956,13 +970,15 @@ tfw_http_req_process(TfwConnection *conn, struct sk_buff *skb, unsigned int off)
 		case TFW_BLOCK:
 			TFW_DBG2("Block invalid HTTP request\n");
 			TFW_INC_STAT_BH(clnt.msgs_parserr);
+			tfw_http_conn_msg_free((TfwHttpMsg *)req);
```

Comment:
This is not essential, but the rest of the code has the following sequence is case of an error:
* send an error message if necessary;
* free the erroneous message;
* increment statistics counters;

New changes make this a different sequence which is contrasting.